recipe/meta.yaml

```diff
@@ -23,16 +23,17 @@ requirements:
     - pip
   run:
     - python >=3.5
+    # requirements.txt
     - pyzmq >=17.1.2
-    - typeguard >=2.5
-    - ipyparallel
+    - typeguard >=2.10
+    - typing-extensions
     - globus-sdk
     - dill
     - tblib
-    - ipykernel
     - requests
     - paramiko
     - psutil >=5.5.1
```

Comment:
These changes should have already happened for the previous version (1.1.0) in #6.  I'm not sure how I missed them.

Reply:
I think I was looking at changes in `setup.py` and not `requirements.txt`.  I'll try to keep better track next time.  Hopefully the comments I added this time will help.

Reply:
These changes look good. I see that there is a section for the optional packages required for the `monitoring` optional_extra, but not for the other extras listed. Was there a reason for including just monitoring?

Reply:
I don't believe I was the one who added those.  I'll look into when it happened and if there was any discussion.

Typically in other feedstocks, we build separate conda packages that add the extras (e.g. `parsl-with-monitoring`).  That would be possible for `parsl`, too.  It's just considerably more complicated to set up.  Let me know if that's of interest.

Reply:
It looks like the `monitoring` dependencies were added here https://github.com/conda-forge/parsl-feedstock/pull/1/commits/a4d9d8a362dd1b69ded5cc8621ffe5db1b6ac90f by @ZhuozhaoLi.  @ZhuozhaoLi, any comment on why those were included and whether we want to include more extras in the main `parsl` package or add `parsl-with-monitoring`, etc.?

Reply:
Here's an example of a feedstock that breaks the extras into their own outputs in the way I'm talking about:
https://github.com/conda-forge/airflow-feedstock/blob/master/recipe/meta.yaml

Reply:
These monitoring dependencies should sit in `parsl-with-monitoring` package. I was not sure how to add a separate package at that time.